kofta/src/vscode-webview/modules/room-chat/RoomChatInput.tsx

```diff
@@ -57,10 +60,14 @@ export const RoomChatInput: React.FC<ChatInputProps> = ({}) => {
     if (changeToIndex !== null)
       setActiveUsername(queriedUsernames[changeToIndex]?.id);
   }
+  const [isEmoji, setisEmoji] = useState(false);
+  const addEmoji = (emoji: any) => {
+    setMessage(message + emoji.native);
```

Comment:
This logic makes it so the emoji is always appended to the end of the message. In my opinion it should be placed wherever the cursor is. `inputRef.current.selectionEnd`

Reply:
ok